### DISCUSSION TITLE
What is the base path of Webserver?

### DISCUSSION THREAD

**Discussion by pmffromspace** [danielalvsaaker/tf-viewer-net#13] [General]
Hey Im able to start the container no problem, but when I open the url, the page is empty and a 404 error is thrown

```

info: Microsoft.AspNetCore.Hosting.Diagnostics[16]
      Request reached the end of the middleware pipeline without being handled by application code. Request path: GET http://192.168.178.2:8080/, Response status code: 404

```

**Comment by danielalvsaaker** [comment]
Great that you got it working! There are however some missing pieces as of now. The base path is `/graphql`, which is only the API endpoint. You would need to set up OpenID Connect authentication first before you can use the API. I will document how to achieve this.

In terms of the frontend, I'm working on it in parallel and the current working state will be available shortly.
